Packs/CortexXDR/.pack-ignore

```diff
@@ -119,4 +119,7 @@ ignore=IF115
 ignore=IF115
 
 [file:incidentfield-XDR_Risky_Host_Count.json]
-ignore=IF115
\ No newline at end of file
+ignore=IF115
+
+[file:InvestigationVerdict.yml]
+ignore=BA124
```

Comment:
Why?

Reply:
"Missing InvestigationVerdict_test.py unit test."
'InvestigationVerdict' is a dynamic section automation.